examples/cluster/cluster.json

```diff
@@ -0,0 +1,51 @@
+{
+    "AWSTemplateFormatVersion": "2010-09-09",
+    "Description": "This template creates a cluster on the MongoDB Atlas API, this will be billed to your Atlas account.",
+    "Parameters": {
+        "ProjectID": {
+            "Type": "String"
+        },
+        "ClusterName": {
+            "Type": "String"
+        }
+    },
+    "Mappings": {},
+    "Resources": {
+        "Cluster": {
+            "Type": "MongoDB::Atlas::Cluster",
+            "Properties": {
+                "ProjectId": {
```

Comment:
I meant to add another resource for `Project` and asks a name for a project then get his id so it can work from scratch

Reply:
Got it, I'll add it.